nevergrad/optimization/base.py

```diff
@@ -149,6 +150,14 @@ def _rng(self) -> np.random.RandomState:
         """
         return self.parametrization.random_state
 
+    @property
+    def pareto_front_extractor(self) -> str:
+        return self.__pareto_front_extractor
+
+    @pareto_front_extractor.setter
+    def pareto_front_extractor(self, pareto_front_extractor: str):
+        self.__pareto_front_extractor = pareto_front_extractor
+
```

Comment:
why do we need this? this seems very weird to me because:
- it adds an API
- it uses setter and getter but without any side effect, so it should only be an instance attribute instead
- it uses double `_` which I never ever use because it usually does not do what you expect when subclassing :)

Reply:
ok I simplify thx.